### PR TITLE
RTCSctpTransportStats with smoothedRoundTripTime added.

### LINKED ISSUE #376: [DataChannels] Use RTT from sctp in the stats
SCTP keeps an RTT parameter which is constantly being adjusted as data is being sent through the channel. Using that RTT can be significantly more accurate than using the ICE heartbeeting. Furthermore, browsers (such as Firefox) would find it easier to implement for data channel only peer connections. @nils-ohlmeier

### PR DESCRIPTION
**Pull request by henbos**
Fixes #376.

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -3463,6 +3474,40 @@ <h2>
...
+            <dl data-link-for="RTCSctpTransportStats" data-dfn-for="RTCSctpTransportStats" class=
+            "dictionary-members">
+              <dt>
+                <dfn><code>smoothedRoundTripTime</code></dfn> of type
+                <span class="idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  The latest smoothed round-trip time value, corresponding to
+                  spinfo_srtt defined in [[RFC6458]] but converted to seconds.
```
Comment (henbos) [inline webrtc-stats.html]: @vr000m When you review can you take a look at the relevant RFCs if there is something else that we could reference from the public SCTP API?

Comment (alvestrand) [reply webrtc-stats.html]: https://tools.ietf.org/html/rfc4960#section-6.3.1 gives the calculation of SRTT.

**Comment by alvestrand** [comment]
Is there implementor interest to add this?
If someone's committed to implement it, we should merge it.

**Comment by henbos** [comment]
@shacharz and @lgrahl can you review and comment?

File: webrtc-stats.html (hunk 1 above)
Comment (vr000m) [reply webrtc-stats.html]: This will report the instantaneous value of SRTT when getStats is called.

**Comment by lgrahl** [comment]
I don't really understand why exactly SRTT is being chosen and none of the other stats but I don't mind.

> Is there implementor interest to add this?

That would be my question, too. /cc @steely-glint

**Comment by alvestrand** [comment]
As to "why SRTT": Bug #376 was specifically about SRTT, and discussion of "what is useful to expose" has mostly been saying "RTT is useful". More stats have costs (slower getStats calls, more code to maintain), so we'd like to make sure there's a benefit to adding a stat before we add it. Separate bugs are good.

**Comment by vr000m** [comment]
Since there was interest, we will go ahead an merge this. We do not foresee implementation issues with this. If there may be a need for more guidance, we can open a new ticket.

File: webrtc-stats.html (hunk 1 above)
Comment (shacharz) [reply webrtc-stats.html]: @henbos 
sorry missed the mention.
converted to seconds means it'll be a fracture?
as in `25`ms will be converted to `0.025`s ?

**Pull request merged**